page-home.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/*
+Template Name: Homepage
+*/
+get_header(); ?>
+
+<main id="primary" role="main" class="content-area">
+	<?php
+		global $post;
```

Comment:
Can we possible add this into a function and return the home_banner? 
1. Doing that we will keep the php and html separated
2. Unit test it